usecases/sharding/state.go

```diff
@@ -260,28 +260,28 @@ func (s *State) initPhysical(names []string, replFactor int64) error {
 
 	s.Physical = map[string]Physical{}
 
+	shards := make(map[string]bool)
```

Comment:
i would rename it `nodeHasShard` or something similar to make clearer the usage of it.

Reply:
I agree. This could be part of the next refactor. For now, I suggest postponing it to avoid triggering the pipeline, as it is not related to the behavior